applications/pivot/src/pivot_call.erl

```diff
@@ -525,6 +527,9 @@ process_resp(Call, CT, RespBody) ->
             {'stop', Call};
         'throw':{'error', 'unrecognized_cmds'} ->
             lager:info("no translators recognize the supplied commands: ~s", [RespBody]),
+            wapi_pivot:publish_failed(RequesterQ, [{<<"Call-ID">>,whapps_call:call_id(Call)}
```

Comment:
Hey, sorry to be picky. (I have to be while @jamesaimonetti is away)
Could you add a space after your comma?